nixos/modules/flyingcircus/roles/nginx.nix

```diff
@@ -43,9 +43,9 @@ let
           "~(?P<ip>.*)" $ip;
       }
 
-      # same as 'nonanonymized'
+      # same as 'anonymized'
       log_format main
```

Comment:
Dunno what all of the nginx stuff is going to do here.

_Please_ check the diff before submitting.